vnext/local-cli/generator-windows/templates/cs/proj/MyApp.sln

```diff
@@ -46,6 +46,7 @@ Global
 		..\node_modules\react-native-windows\Microsoft.ReactNative.Cxx\Microsoft.ReactNative.Cxx.vcxitems*{da8b35b3-da00-4b02-bde6-6a397b3fd46b}*SharedItemsImports = 9
 		..\node_modules\react-native-windows\Chakra\Chakra.vcxitems*{f7d32bd0-2749-483e-9a0d-1635ef7e3136}*SharedItemsImports = 4
 		..\node_modules\react-native-windows\JSI\Shared\JSI.Shared.vcxitems*{f7d32bd0-2749-483e-9a0d-1635ef7e3136}*SharedItemsImports = 4
+		..\node_modules\react-native-windows\Microsoft.ReactNative.Cxx\Microsoft.ReactNative.Cxx.vcxitems*{f7d32bd0-2749-483e-9a0d-1635ef7e3136}*SharedItemsImports = 4
```

Comment:
FYI, this is a breaking change, since users will need to edit their existing solution.

Reply:
Actually all of the Sln edits point to existing vcxitems but with a different GUID? Do you know what happened here?

Reply:
Its a change the VS seems to fixup automatically, so I'm not too worried.